### ISSUE TITLE
Extension causes high cpu load

### ISSUE THREAD

**Issue by carun** [Pure-D/code-d#325]
- Issue Type: `Performance`
- Extension Name: `code-d`
- Extension Version: `0.22.0`
- OS Version: `Windows_NT x64 10.0.19041`
- VSCode version: `1.47.3`

:warning: Make sure to **attach** this file from your *home*-directory:
:warning:`C:\Users\217216x715132\webfreak.code-d-unresponsive.cpuprofile.txt`

Find more details here: https://github.com/microsoft/vscode/wiki/Explain-extension-causes-high-cpu-load

**Comment by carun** [comment]
To make it clear, I wasn't doing anything at all during high CPU usage. I just fired up VSCode and let it open my D project.

**Comment by cyrusmsk** [comment]
I faced the same problem. Actually there are not so many extentions of VCS for Dlang. And currently I need linter for my code. So I will really appreciate if someone could recommend any other working extention for that purpose.

**Comment by dd86k** [comment]
Yep just had the same message this morning, and it's a vibe-d project.

| Module | Version |
|---|---|
| OS | Windows 10 20H2 (10.0.19042.685) AMD64 |
| VSCodium | 1.52 (user) |
| code-d | 0.22.0 |
